slf4j-simple dependency causing issues

I the Spring Boot project I work on I have an encountered the following issue after adding the `pl.pojo:pojo-tester:0.7.3` dependency:

`Caused by: java.lang.IllegalArgumentException: LoggerFactory is not a Logback LoggerContext but Logback is on the classpath. Either remove Logback or the competing implementation (class org.slf4j.impl.SimpleLoggerFactory loaded from file:~/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-simple/1.7.22/595058a6ec00a6e58f2f48113e8708f036a19cf7/slf4j-simple-1.7.22.jar). If you are using WebLogic you will need to add 'org.slf4j' to prefer-application-packages in WEB-INF/weblogic.xml Object of class [org.slf4j.impl.SimpleLoggerFactory] must be an instance of class ch.qos.logback.classic.LoggerContext`

I have created a sample project exposing this problem: https://github.com/adamsiemion/springboot-with-pojo-tester

After a quick investigation I found out that `pojo-tester` has a dependency to `slfj4-simple` and changing:

    testCompile ('pl.pojo:pojo-tester:0.7.3')

to

    testCompile ('pl.pojo:pojo-tester:0.7.3') { 
      exclude group: "org.slf4j", module: "slf4j-simple" 
    }

fixes the issue.

Since `pojo-tester` is a library I think it should not add a logger implementation. Instead it should  has a dependency only to the logger interfaces (`slfj4-api`).

BTW. I think the library is cool and I use it in my project :)

--- a/build.gradle
+++ b/build.gradle
@@ -81,7 +81,7 @@ junitPlatform {
 }
 
 dependencies {
-    compile("org.slf4j:slf4j-log4j12:1.7.22")
+    compile("org.slf4j:slf4j-api:1.7.24")
     compile("org.apache.commons:commons-lang3:3.4")
     compile("org.apache.commons:commons-collections4:4.1")
     compile("com.googlecode.combinatoricslib:combinatoricslib:2.1")

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,6 +1,6 @@
-#Wed Mar 08 18:23:02 CET 2017
+#Wed Mar 08 20:22:19 CET 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.4-all.zip

